csharp/extractor/Semmle.Extraction.CSharp/Entities/EventAccessor.cs

```diff
@@ -53,14 +54,14 @@ public override void Populate(TextWriter trapFile)
             Overrides(trapFile);
         }
```

Comment:
Properties have the following too. Shouldn't it be added?

```
            if (Symbol.FromSource() && Block is null)
            {
                trapFile.compiler_generated(this);
            }
```

Reply:
Makes sense.